src/main/java/org/jitsi/jicofo/recording/jibri/JibriSession.java

```diff
@@ -411,7 +408,7 @@ private void sendJibriStartIq(final Jid jibriJid)
         try
         {
             JibriIq result = (JibriIq)xmpp.sendPacketAndGetReply(startIq);
-            setJibriStatus(result.getStatus(), null);
+            handleJibriStatusUpdate(jibriJid, result.getStatus(), result.getFailureReason());
```

Comment:
I'm not sure I remember things correctly, but isn't it the case that the `result` might be `null` in case of a timeout ? or will that trigger OperationFailedException ? then in case of a timeout we won't try another one neither we would mark the one that is not responding as unhealthy in any way, right ?

Reply:
i assume we'll get the exception--at least, if there's an issue there we're no worse off than we were before (and i don't believe we've seen any NPE here due to that).  the pending timeout would come in and clean things up.

Reply:
It might be worth trying another one if the first one refuses to respond, but I guess we can do this incrementally, so I guess it's fine this way for now.